matlab_simulation/08-planning/example_search_graph.m

```diff
@@ -0,0 +1,47 @@
+% Get same problem every time (fix random number generator start point)

```

Comment:
Consider adding a line at the top that tells the users what to expect and what parameters to change in order to test their own examples